Makefile

```diff
@@ -314,16 +317,22 @@ run: generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: ## Build docker image with the manager.
-	docker build -t $(IMG) .
+	$(CONTAINER_ENGINE) build -t $(IMG) .
 
 docker-push: ## Push docker image with the manager.
-	docker push $(IMG)
+	$(CONTAINER_ENGINE) push $(IMG)
 
 kind-load-image: ## Load image to local cluster
-	$(KIND) load docker-image $(IMG) --name $(KIND_CLUSTER_NAME)
+	@mkdir -p ./tmp
+	@if [ -f ./tmp/image.tar ]; then rm ./tmp/image.tar; fi
+	$(CONTAINER_ENGINE) save -o ./tmp/image.tar $(IMG)
+	KIND_EXPERIMENTAL_PROVIDER=$(CONTAINER_ENGINE) $(KIND) load image-archive ./tmp/image.tar $(IMG) --name $(KIND_CLUSTER_NAME)
 
 kind-load-bundle: ## Load image to local cluster
-	$(KIND) load docker-image $(BUNDLE_IMG) --name $(KIND_CLUSTER_NAME)
+	@mkdir -p ./tmp
```

Comment:
AFAIK make target `kind-load-bundle` is not being used. And if it was being used, I would not DRY. instead:

```make
kind-load-bundle: ## Load image to local cluster    
    $(MAKE) kind-load-image IMG=$(BUNDLE_IMG)
```

I would remove the target if not being used.